.github/workflows/ios.yml

```diff
@@ -128,7 +135,7 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         uses: softprops/action-gh-release@v1
         with:
-          prerelease: endsWith(github.ref, '-beta')
+          # prerelease: endsWith(github.ref, '-beta')
```

Comment:
why did you put this line into a comment?

Reply:
This was a test to check the `pre-release` flag on https://github.com/mapswipe/mapswipe/releases. I noticed that the android build was setting it correctly, but then the ios build (which always takes much longer) was switching it incorrectly, this was to check that my impression was right (it is). Now I need to figure out a fix :)

Reply:
should be ok now.